src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] map; // наш масив
+    private int size = 0; // кількість елементів
+    private double threshold; // значення критичного заповненення масиву
+
+    public MyHashMap() {
+        map = (Node<K, V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
+    }
+
+    private static class Node<K,V> {
+        V value;
+        MyHashMap.Node<K,V> next;
+        private int hash;
+        private K key;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        Node<K, V> newNode = new Node<>(key, value, null);
+        int index = getIndex(key); // знаходимо індекс
```

Comment:
```suggestion
        int index = getIndex(key);
```
if you write comments - use eng please. 
But try to use comments only when your code behaves unpredictably

Reply:
done